android/app/src/main/res/layout/quest_recycler_view_item.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@drawable/border"
+    android:paddingHorizontal="5dp"
+    android:paddingVertical="15dp"
+    android:clickable="true"
+    android:focusable="true"
+    android:gravity="center">
+
+    <ImageView
+        android:id="@+id/quest_image_view"
+        android:layout_width="70dp"
+        android:layout_height="70dp"
+        android:layout_marginTop="0dp"
+        android:layout_marginStart="0dp"/>
+
+    <TextView
+        android:id="@+id/quest_name_text_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="0dp"
+        android:layout_marginTop="00dp"
+        android:layout_toEndOf="@id/quest_image_view"
+        android:gravity=""
+        android:layout="@id/quest_image_view"
+        android:textSize="20sp" />
+
+    <TextView
+        android:id="@+id/quest_owner_text_view"
+        android:layout_width="wrap_content"
+        android:layout_height="30dp"
+        android:layout_below="@id/quest_name_text_view"
+        android:layout_marginStart="0dp"
+        android:layout_marginTop="0dp"
+        android:layout_toEndOf="@id/quest_image_view"
+        android:gravity=""
+        android:textSize="17sp" />
+
+    <TextView
+        android:id="@+id/quest_exp_text_view"
+        android:layout_width="wrap_content"
+        android:layout_height="30dp"
+        android:layout_below="@id/quest_name_text_view"
+        android:layout_marginStart="30dp"
+        android:layout_marginTop="0dp"
+        android:layout_toEndOf="@id/quest_owner_text_view"
+        android:gravity=""
+        android:textSize="17sp" />
+
+    <TextView
+        android:id="@+id/quest_mandatory_text_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/quest_image_view"
+        android:layout_marginTop="5dp"
+        android:textSize="17sp"
+        android:layout_marginBottom="10dp"
+        android:layout_marginStart="0dp"/>
```

Comment:
is this needed?

Reply:
xml is real messy rn, I was gonna fix the layout later but I can clean up a bit